src/styles/components/Header.scss

```diff
@@ -0,0 +1,6 @@
+.header {
+  &__logo {
+    max-width: 12rem;
+    width: 100%;
+  }
+}
```

Comment:
I'd like us to follow BEM conventions a little more, so I would rename this class to `.cmp-header`. The file should also start with an underscore to indicate it as a SCSS partial, so `_header.scss`. Then this should be imported in `globals.scss` instead of the `_app.jsx` file.

```scss
// globals.scss
// we don't need to include the underscore or the file extension. sass will figure out the right file
@import "./components/header";
```